src/Schema/Visitor/Graphviz.php

```diff
@@ -83,7 +83,7 @@ private function createTableLabel(Table $table): string
                 . '</TD>'
                 . '<TD BORDER="0" ALIGN="LEFT" BGCOLOR="#eeeeec">'
                 . '<FONT COLOR="#2e3436" FACE="Helvetica" POINT-SIZE="10">'
-                . strtolower($column->getType()->getName())
+                . get_class($column->getType())
```

Comment:
This is sort of a BC break.

Reply:
Yeah… kinda.

Reply:
We can leave this code as is and completely remove it in 4.0.x.

Reply:
For reference, it was removed in https://github.com/doctrine/dbal/pull/5099.